docs/install.md

```diff
@@ -71,10 +71,10 @@ More info can be found on the wcstools <a href="http://tdc-www.harvard.edu/wcsto
 
 ### Install DRESSCode
 
-Requires Python >= 3.7. Make sure you have a working python environment then install from source:
+Requires Python >= 3.7. Make sure you have a working python environment (it is probably best to create a new environment to work with DRESSCode). Then install from source:
 
 ```cmd
-python -m pip install https://git+https://github.com/spacetelescope/DRESSCode.git
+pip install git+https://github.com/spacetelescope/DRESSCode
```

Comment:
we should keep the `python -m` part

Reply:
it is not needed, but is best practice. if you have `pip` installed in a weird way, it'll run whatever that version is on your path. if you call it as a python module (with the python -m pip) then it will run in your current python environment